rules/DeadCode/Rector/Property/RemoveUnusedPrivatePropertyRector.php

```diff
@@ -111,29 +110,11 @@ public function refactorWithScope(Node $node, Scope $scope): ?Node
             );
 
             if ($isRemoved) {
-                $this->processRemoveSameLineComment($node, $property, $key);
-                $hasRemoved = true;
+                $hasChanged = true;
```

Comment:
comment on line 103 need to be updated `$hasRemoved` -> `$hasChanged`

Reply:
Thanks :+1: